Telemetry.md

```diff
@@ -0,0 +1,63 @@
+
+### Telemetry
+
+WrenAI relies on anonymous usage statistics to continuously improve. That's why we ask you to share your usage data with us.
+WrenAI use [Posthog](https://posthog.com/) as out telemetry tool
+
+
+### How Does Telemetry help WrenAI
+
+Based on the telemetry, we can know which features are frequently used and related to our community or in which step an error occured and block our user.
+We can use these information to imporve our feature and as a reference for deciding on the roadmap.
+
+
+### What Information Is Shared
+
+Telemetry in WrenAI comprises anonymous usage statistics of basic components, such as `DataSource`, `Preview SQL`, `Ask` or any other component.
+We receive an event every time these components are initialized or used.
+Each event contains an anonymous, randomly generated user ID (uuid) and a collection of properties about your execution environment. They never contain properties that can be used to identify you, such as: `IP addresses`, `Current OS username`, `File paths`, ...
+...
+
+Here is an example event that we will collect:
+```
+# example to an "server start" event
+{
+  "uuid": "018eeb03-85db-73be-b7b6-84b2d735f19c", # the event id
+  "event": "server_start", 
+  "properties": {
+    "wren-ui-version": "telemetry",
```

Comment:
values of wren-ui-version and wren-engine_version are strange. also there is underscore in `wren-engine_version`; while there is no underscore in `wren-ui-version`
 
  "wren-ui-version": "telemetry",
  "wren-engine_version": null,